rustls/src/common_state.rs

```diff
@@ -186,6 +187,49 @@ impl CommonState {
         self.send_plain(data, Limit::Yes)
     }
 
+    pub(crate) fn eager_send_some_plaintext(
```

Comment:
I didn't love the `send_some_plaintext()` name and `eager_` is even worse (since it's longer). What is eager about it? The name should clearly explain how it's different from `send_some_plaintext()`. Maybe it's `buffer_plaintext()` vs `write_plaintext()`?